charts/seed-bootstrap/charts/elastic-kibana-curator/templates/elasticsearch/elasticsearch-vpa.yaml

```diff
@@ -9,4 +9,4 @@ spec:
     kind: StatefulSet
     name: elasticsearch-logging
   updatePolicy:
-    updateMode: "Off"
+    updateMode: "Auto"
```

Comment:
To make VPA for elastic to work, these flags must be adapted, too. 

https://github.com/gardener/gardener/blob/f85b9461857c5e6652c2b3258d57cf5bbe1e4fd7/charts/seed-bootstrap/charts/elastic-kibana-curator/templates/elasticsearch/statefulset.yaml#L47

Reply:
I have spent some time on this.
The used elastic is running on java 12
```bash
$ /opt/jdk-12/bin/java  -version                                                                                     
openjdk version "12" 2019-03-19
OpenJDK Runtime Environment (build 12+33)
OpenJDK 64-Bit Server VM (build 12+33, mixed mode, sharing)
```

For this version `UseContainerSupport` is enabled by default, see https://www.eclipse.org/openj9/docs/xxusecontainersupport/

To achieve better performance, I think we have to replace the `-Xms` and `-Xmx` flags with the more container friendly `InitialRAMPercentage`, `MaxRAMPercentage` and `MinRAMPercentage`, ref https://www.eclipse.org/openj9/docs/xxinitialrampercentage/. Or let use the default ones calculated on the detected resources, but remove the `-Xms` and `-Xmx`, for sure.

Ref:
https://merikan.com/2019/04/jvm-in-a-container/#java-10

Reply:
@vpnachev thanks for digging into this. I've removed the flags. Can you check again, please?

Reply:
Unfortunately, this is not enough. 
You have to remove them from here, too. https://github.com/gardener/gardener/blob/cc4dd233de46aff4f69b3aedffcb91817e766e16/charts/seed-bootstrap/charts/elastic-kibana-curator/templates/elasticsearch/configmap.yaml#L129-L130

Reply:
Thanks, done.